app/src/main/java/com/mycompany/myapp/ui/main/MainViewModel.kt

```diff
@@ -43,7 +42,7 @@ class MainViewModel @Inject constructor(
         fetchCommits()
     }
 
-    private var commits: Commits = Commits.Result(emptyList())
+    internal var commits: Commits = Commits.Result(emptyList())
```

Comment:
@patrickhammond Had to change the scope of this guy for testing purposes.  I needed a handle to `commits` while unit testing the `fetchCommits()` method.  Not too happy about it but curious what you think.

Reply:
That seems 100% legit to me. I've started defaulting to `internal` anywhere I'd have package scope in Java. For better or for worse. @weverb2 any guidance or opinions you have on best practices around Kotlin access modifiers wrt to supporting testing?

Reply:
@atoennis the only thing I would have to say is maybe mark it `@VisibleForTesting`?

Reply:
Done!